xml_inscriptions/bib_only/MA.Camb.HU.Sack.L.1980.83.xml

```diff
@@ -63,6 +63,10 @@
                   <ptr type="rest-of-bibl" target="#ILS"/>
                   <biblScope unit="item">9433</biblScope>
                </bibl>
+               <bibl>
+                  <ref type="external"
+                     target="http://www.edr-edr.it/edr_programmi/res_complex_comune.php?do=book&amp;id_nr=EDR079159&amp;fo_antik=&amp;fo_modern=&amp;fundstelle%5B%5D=&amp;boolfundstelle=AND&amp;fundstelle%5B%5D=&amp;aufbewahrung%5B%5D=&amp;boolaufbewahrung=AND&amp;aufbewahrung%5B%5D=&amp;denkmaltyp=&amp;booldenkmaltyp=OR&amp;denkmaltyp2=&amp;ope_mis_alto=%3C%3D&amp;mis_alto=&amp;ope_mis_largo=%3C%3D&amp;mis_largo=&amp;ope_mis_spesso=%3C%3D&amp;mis_spesso=&amp;boollivelloSocialeTot=OR&amp;Bibliografia%5B%5D=&amp;boolBibliografia=AND&amp;Bibliografia%5B%5D=&amp;Testo=&amp;boolTesto=AND&amp;Testo2=&amp;Apparato%5B%5D=&amp;boolApparato=AND&amp;Apparato%5B%5D=&amp;dat_tag=&amp;dat_monat=&amp;ope_dat_jahr_a=%3D&amp;dat_jahr_a=&amp;ope_dat_jahr_e=%3D&amp;dat_jahr_e=&amp;periodizzazione=&amp;bool=AND&amp;ordinamento=id_nr&amp;javasi=javascriptsi&amp;se_foto=tutte&amp;lang=en">EDR079159</ref>
```

Comment:
Generally speaking try to prefer the shorter version of the URL, if possible (just through the id_nr).